db/seeds.rb

```diff
@@ -3,9 +3,19 @@
     AdminUser.create(email: 'admin@example.com', password: 'password', password_confirmation: 'password')
   end
 
-  french_press = Device.find_or_create_by(name: 'French Press', image_url: 'https://cdn.shopify.com/s/files/1/0219/5410/files/french_press_icon.jpg?1078')
-  chemex = Device.find_or_create_by(name: 'Chemex', image_url: 'https://images-na.ssl-images-amazon.com/images/I/410XbJTPshL._SY355_.jpg')
-  keurig = Device.find_or_create_by(name: 'Keurig', image_url: 'https://allaboutharts.files.wordpress.com/2015/04/keurig_logo_highres.jpg')
+  device_image = Photo.create(name: 'Device Image') do |photo|
+    photo.image = File.open(Rails.root.join("assets/images/device.jpg"))
+  end
+  roast_image = Photo.create(name: 'Roast Image') do |photo|
+    photo.image = File.open(Rails.root.join("assets/images/roast.jpg"))
+  end
+
+  french_press = Device.find_or_create_by(name: 'French Press')
+  chemex = Device.find_or_create_by(name: 'Chemex')
+  keurig = Device.find_or_create_by(name: 'Keurig')
+  french_press.image = device_image
+  chemex.image = device_image
+  keurig.image = device_image
```

Comment:
You need to save each of these devices.  Right now, you're setting the image property on them, but nothing is being saved to the database.